.github/workflows/build.yml

```diff
@@ -410,6 +410,7 @@ jobs:
         board:
         - "espressif_saola_1_wroom"
         - "espressif_saola_1_wrover"
+        - "micro_s2" 
```

Comment:
Please rename the board name to include your brand as well as the board name so `microdev_micro_s2` or something similar. I'm expecting many S2 boards and `micro_s2` is pretty generic on its own.

Reply:
Thanks for your suggestions. I made the necessary changes to include microDev in the board name.